app/frames/waitlists/[slug]/join/route.tsx

```diff
@@ -339,7 +340,31 @@ const frameHandler = frames(async (ctx) => {
             >
               Try again
             </Button>,
-            <Button action="link" key="1" target={waitlist.externalUrl}>
+            requiredChannelsList[0] ? (
+              <Button
```

Comment:
Are you sure that this doesn't give any error if requiredChannelsList is undefined or empty

Reply:
Pretty sure but I'll double test it!